tripper/mappings/mappings.py

```diff
@@ -220,7 +220,7 @@ def __init__(
 
     def add_inputs(self, inputs: "Inputs") -> None:
         """Add input dict for an input route."""
-        assert isinstance(inputs, dict)  # nosec B101
+        assert isinstance(inputs, Mapping)  # nosec B101
```

Comment:
We are only using the functionality expected by a mapping (key item access), so it shouldn't cause any problems (unless we have more explicit tests for dict rather than mapping). 

This function is mainly intended for internal use by the mappings module. Otherwise, we should really have documented better how the `inputs` argument should be structured.

Reply:
OK. Thank you!